.github/workflows/testing.yml

```
@@ -0,0 +1,87 @@
+name: CI
+on:
+  - push
+  - pull_request
```

```suggestion
  - pull_request:
    branches:
      - '*'
      - '!master'
```
to prevent running two similar lints when open pull request in master